app-backup/mkstage4/mkstage4-0.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Bash Utility for Creating Stage 4 Tarballs"
+HOMEPAGE="https://github.com/TheChymera/mkstage4"
+SRC_URI="https://github.com/TheChymera/mkstage4/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	app-arch/tar
+	app-shells/bash
+"
+
```

Comment:
tar and bash are in default @system so you could just remove RDEPEND per https://devmanual.gentoo.org/general-concepts/dependencies/index.html#implicit-system-dependency

Reply:
This is true, but some people want to add them to imply it's a simple bash script. As it's not a strict policy, I leave it up for the maintainer. I'd personally remove them.